android/app/build.gradle

```diff
@@ -130,7 +130,7 @@ android {
         abi {
             reset()
             enable enableSeparateBuildPerCPUArchitecture
-            universalApk false  // If true, also generate a universal APK
+            universalApk true  // If true, also generate a universal APK
```

Comment:
Is a universal APK necessary anyways? To my knowledge these days there are very few (no?) mobile devices on non-ARM architectures?

Reply:
veeeeery few, probably never needed to be honest .. until someone has one

Reply:
![clever](https://user-images.githubusercontent.com/43223637/63230851-eb33bd00-c266-11e9-959f-ed2776f05bae.jpg)

Reply:
![cleverthinking](https://user-images.githubusercontent.com/43223637/63230868-4c5b9080-c267-11e9-8113-f7cbd93fc886.jpg)